python/mlc_chat/chat_module.py

```diff
@@ -574,6 +574,23 @@ def generate(self):
         # TODO: work in progress
         pass
 
+    def embed_text(
+        self,
+        input: str,
+        place_in_prompt: PlaceInPrompt = PlaceInPrompt.All
```

Comment:
Should we make this `PlaceInPrompt.Middle` by default?

Reply:
sg! changed